__integrations__/__mocks__/fixtures.ts

```diff
@@ -339,6 +346,28 @@ export const fillServices = async (
   )();
 };
 
+export const fillProfiles = async (
+  db: Database,
+  profiles: ReadonlyArray<ProfileCollection.Profile>
+): Promise<void> => {
+  await pipe(
+    db.container(ProfileCollection.PROFILE_COLLECTION_NAME),
+    TE.of,
+    TE.map(p => new ProfileCollection.ProfileModel(p)),
```

Comment:
A more compact version:
```suggestion
    p => TE.of(new ProfileCollection.ProfileModel(p)),
```
I see the same pattern on row 376